app/controllers/merchant/orders_controller.rb

```diff
@@ -1,5 +1,9 @@
+require './app/models/poros/merchant_show'
```

Comment:
Not familiar with this, curious as to what it is doing exactly.

Reply:
This is how it knows about the PORO